src/ctypes/ldouble.mli

```diff
@@ -0,0 +1,258 @@
+(*
+ * Copyright (c) 2016 Andy Ray.
+ *
+ * This file is distributed under the terms of the MIT License.
+ * See the file LICENSE for details.
+ *)
+
+type t
+(** The type of long doubles. *)
+
+val to_float : t -> float 
+(** Convert a long double to a float *)
+
+val of_float : float -> t 
+(** Create a long double from a float *)
+
+val to_int : t -> int 
+(** Convert a long double to an int *)
```

Comment:
This should document the behaviour on overflow -- ideally to raise an exception.